ClassLibraries/Macross.Json.Extensions/Test/DateTimeWireFormatHelperTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Text;
+using System.Text.Json.Serialization;
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Macross.Json.Extensions.Tests
+{
+	[TestClass]
+	public class DateTimeWireFormatHelperTests
+	{
+		[TestMethod]
+		[DataRow("/Date(1)/", 1)]
+		[DataRow("/Date(-1)/", -1)]
+		[DataRow("/Date(18+1234)/", 18, 1, 12, 34)]
+		[DataRow("/Date(-20-9876)/", -20, -1, 98, 76)]
+		[DataRow(@"\/Date(1580803200000)\/", 1580803200000)]
```

Comment:
Should we add one more valid sterilization tested somewhere? This is the one causing issues for us. Thanks!

Date(-62135575200000-0600)